scripts/ci/run-test-microbench.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+export TESTCMD=`dirname "${BASH_SOURCE[0]}"`/run-step.sh
+
+${TESTCMD} --label=microbenchmark --timeout=10m make -C mono/tests/microbenchmarks test-BinaryTrees
```

Comment:
Can I first see a plan what we are going to do with this? The tests will be run on heavily loaded machines with VM config which can change anytime which will make the results very hard to reproduce.

Reply:
My use for this is to profile the runtime and to triage runtime and codegen issues that cost us performance. 

From a CI/QA perspective, the main use for this suite is to compare two commits against each other for performance on some metric.

My main use for this will be in-tree though. I made sure to make it easy to run this without triggering a rebuild for that reason.